external/runtime/runtime.depproj

```diff
@@ -34,6 +34,9 @@
       <!-- Following is version used in release/uwp6.0 -->
       <Version Condition="'$(TargetGroup)'=='uap10.0.16299'">2.1.0-b-uwp6-25707-02</Version>
     </PackageReference>
+    <PackageReference Include="Microsoft.NETCore.ILAsm">
```

Comment:
This also needs to be set here as a side-effect of how CoreFX does restore. If you restore/build the `ilproj` directly, this isn't needed.